estate/models/estate_property.py

```diff
@@ -25,3 +29,15 @@ class EstateProperty(models.Model):
             ("west", "West"),
         ]
     )
+    active = fields.Boolean(default=True)
+    status = fields.Selection(
+        [
+            ("new", "New"),
+            ("offer_received", "Offer Received"),
+            ("offer_accepted", "Offer Accepted"),
+            ("sold", "Sold"),
+            ("canceled", "Canceled"),
+        ],
+        default="new",
+        copy=False,
```

Comment:
```suggestion
    status = fields.Selection(
        selection=[
            ("new", "New"),
            ("offer_received", "Offer Received"),
            ("offer_accepted", "Offer Accepted"),
            ("sold", "Sold"),
            ("canceled", "Canceled"),
        ],
        default="new",
        copy=False,
    )
```

Please, always, run pre-commit-vauxoo -t all. 

The exercise says:

 It must be **required**, should not be copied and should have its default value set to ‘New’.